### DISCUSSION TITLE
Way for pushReplacement

### DISCUSSION THREAD

**Discussion by vaidarbhi01** [csells/go_router#292] [Q&A]
Kindly let me know that how to implement `pushReplacement` with this package as `readme` not provide proper explanation for all way of routing. (push , pop , pushReplacement)

**Comment by csells** [comment]
Each call to context.go() replaces the entire stack of pages. Just define a route with the pages you want and navigate to that.

**Reply by abdimussa87** [reply]
But clicking on the back button from the browser sends back to the previous page, which isn't what a push replacement is supposed to do?

**Reply by csells** [reply]
Flutter navigation doesn't control the contents of the browser's back stack. If you'd like to keep locations from going into the brower's back stack, you can: https://gorouter.dev/web-history

**Reply by abdimussa87** [reply]
Understood. Thank you for the amazing work!!
